pages/common/objdump.md

```diff
@@ -7,10 +7,14 @@
 
 `objdump -f {{binary}}`
 
-- Display the dis-assembled output of executable sections:
+- Display the disassembled output of executable sections:
 
 `objdump -d {{binary}}`
 
+- Display the disassembled executable sections in intel syntax:
+
+`objdump -M intel -d {{binary}}`
```

Comment:
```suggestion
`objdump -M {{intel}} -d {{binary}}`
```

Is there multiple options that can go with -M such as amd

Reply:
For x86 there is just Intel or AT&T (default for this and GNU tools). AMD processors are also x86.

Reply:
We should the {{ }} around it since it takes arm I assume.

Reply:
ARM doesn't have two versions of syntax like x86. In my original post I made a note that maybe this should be mentioned this only applies for x86 binaries